recipes/PythonVirgoTools/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pythonvirgotools" %}
+{% set version = "5.1.1" %}
+
+package:
+   name: "{{ name }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: d425eab31582847ea7add85db3adf8d38253c60fbe500f6ffac0eab3d73ee136
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Is the reason for this the lack of `pyfd` on anything but Linux? If so, you can instead do:
```suggestion
  noarch: python
```
and we can ignore the build failures.

---

Yes, both `PyCm` and `PyFd` are available on Linux. Okay done.